pages/en/lb3/readmes/loopback-connector-cloudant.md

```diff
@@ -561,6 +585,71 @@ module.exports = function(server) {
 };
 ```
 
+# Bulk replace
+
+Given an array of data to be updated, Cloudant supports the idea of performing bulk replace on a model instance.
```

Comment:
I am proposing to add a note that bulk replace is not invoking any operation hooks.